pynetworktables2js/tornado_handlers.py

```diff
@@ -24,48 +16,31 @@ class NetworkTablesWebSocket(WebSocketHandler):
         A tornado web handler that forwards values between NetworkTables
         and a webpage via a websocket
     '''
+
+    ntserial = None
     
     def open(self):
         logger.info("NetworkTables websocket opened")
-
         self.ioloop = IOLoop.current()
-        self.nt = NetworkTable.getGlobalTable()
-        NetworkTable.addGlobalListener(self.on_nt_change, immediateNotify=True)
-        self.nt.addConnectionListener(self, immediateNotify=True)
+        self.ntserial = NTSerial(self.send_msg_threadsafe)
 
     def on_message(self, message):
-        # called when message is received from the dashboard
-        data = loads(message)
-        self.nt.putValue(data['k'], data['v'])
-   
-    def on_close(self):
-        logger.info("NetworkTables websocket closed")
-        NetworkTable.removeGlobalListener(self.on_nt_change)
-    
-    def send_to_dashboard(self, msg):
+        if self.ntserial is not None:
```

Comment:
Because ntserial is initialized from the ```open()``` function call, and I encountered scenarios where other methods would get called before ntserial was initialized.